front/src/modules/simulationResult/components/ChartHelpers/drawElectricalProfile.ts

```diff
@@ -1,35 +1,44 @@
 import { pointer } from 'd3-selection';
 import i18n from 'i18n';
+import { Chart } from 'reducers/osrdsimulation/types';
+import { ElectricalConditionSegment, DrawingKeys } from 'applications/operationalStudies/consts';
 import { getElementWidth } from './ChartHelpers';
 
 const drawElectricalProfile = (
-  chart,
-  classes,
-  dataSimulation,
-  groupID,
-  interpolation,
-  keyValues,
-  name,
-  rotate,
-  isStripe,
-  isIncompatible,
-  // TODO : when migrated id should be reverted with null by default
-  id
+  chart: Chart,
+  classes: string,
+  dataSimulation: ElectricalConditionSegment,
+  groupID: string,
```

Comment:
This is the right time to not use generic naming when this is not necessary
electricalConditionSegment: ElectricalConditionSegment